### ISSUE TITLE
Built result will be different when environment change or node_modules be reinstalled

### ISSUE THREAD

**Issue by Taosky** [Kocal/vue-web-extension#686]
**Describe the bug**
Build result will change when reinstall node_modules. 
This will lead to the problem of the source code review when the expanded store is launched.
As I encountered, when upload to firefox addon center, the code-reviewer reminded the code built result differently from the uploaded code.

**To Reproduce**
1. `npm run build`
2. Save production zip file
3. Delete node_modules dir
4. `npm install && npm run build`
5. compare the two production file

**Comment by Taosky** [comment]
<img width="676" alt="图片" src="https://user-images.githubusercontent.com/7688350/179496768-c664eedf-94be-4a16-b24d-af67078a1a07.png">

**Comment by spekulatius** [comment]
Have you been able to identify the diff @Taosky?

**Comment by Taosky** [comment]
> Have you been able to identify the diff @Taosky?

I just used `diff` command for comparison, It may be that some content is in a different order.

**Comment by spekulatius** [comment]
> > Have you been able to identify the diff @Taosky?
> 
> I just used `diff` command for comparison, It may be that some content is in a different order.I asked a friend to test it and the results showed different.

You mean files in a different order in the ZIP file?

> 
> I turned off code compression, obfuscation and other functions through webpack configuration, but the problem still exists.

Okay, interesting. I never had this issue...

> 
> Finally, I switched to [vue-cli-plugin-chrome-extension-cli](https://github.com/sanyu1225/vue-cli-plugin-chrome-extension-cli), it has no such problem.

**Comment by Taosky** [comment]
> You mean files in a different order in the ZIP file?

What I mean is that the location of the final generated JS code in the file may change, or the variable name may change (sorry, I don't know much about the code after the build, in short, after each build, the generated code changed)

In most cases, this change will not cause problems, but mozilla requires that extensions that use webpack upload source code for review. I think they might use something like 'diff' in the background to compare the results.

When uploading to chrome webstore or microsoft store (Edge) there is no problem.

**Comment by spekulatius** [comment]
Yeah, I know Mozilla likes to have deterministic code. It's understandably desirable to have the same output each time, hence my surprise about the issue.

I'm happy you found a solution that works for you.
